### PR TITLE
Django 1.10

### PR DESCRIPTION
**Pull request by dwaynebailey**
This brings Django 1.10 support to Pootle.

The concerns I have are:

* c523ad38608afd0a862d89c7b4f8d1b00a39b77b - not sure this is a correct fix, though it works, needs clicky testing I think

### CODE REVIEW

File: pytest_pootle/factories.py [hunk 1]
```diff
@@ -115,7 +115,6 @@ def fullname(self):
         # returns an incrementing index relative to the tp
         return 'Project %s' % Project.objects.count()
 
-    pootle_path = factory.LazyAttribute(lambda p: "/projects/%s" % p.code)
```
Comment (phlax) [inline pytest_pootle/factories.py]: >  Not needed anymore 

how come?

Comment (dwaynebailey) [inline pytest_pootle/factories.py]: No frigging idea :)

I think this was one we debugged together at PooCon but can't recall the reason

Comment (phlax) [inline pytest_pootle/factories.py]: wfm 8/

**Comment by dwaynebailey** [comment]
Blockers

* Requires and unreleased version of allauth.  To run in Django 1.10 allauth needs these changes https://github.com/pennersr/django-allauth/commit/b63e8db0e276bf3a4c6fe4742f1843d15632b125

**Pull request merged**